dhall/src/Dhall/TypeCheck.hs

```diff
@@ -3037,7 +3098,11 @@ prettyTypeMessage (UnusedHandler ks) = ErrorMessages {..}
 
 prettyTypeMessage (MissingHandler ks) = ErrorMessages {..}
   where
-    short = "Missing handler"
+    short = case Data.Set.toList ks of
+         []       -> "Missing handler"
+         [x]      -> "Missing handler: " <> pretty x
+         xs@(_:_) -> "Missing handlers: " <> (Pretty.hsep . Pretty.punctuate Pretty.comma 
+                                             . map Dhall.Pretty.Internal.prettyLabel $ xs)
```

Comment:
What does this look like?

Reply:
Oops. This got cut out somehow:

```
Error: Missing handlers: `Almost Left`, Middle, MildyConservative

7│     merge handlers union

```

For 
```
     λ(union : < Left : Natural | `Almost Left` : Natural | Middle: Text | MildyConservative : Bool
            |Right : Bool >)
→   let handlers =
            { Left  = Natural/even  -- Natural/even is a built-in function
            , Right = λ(b : Bool) → b
            }
in  merge handlers union
```